package.json

```diff
@@ -68,6 +68,7 @@
     },
     "overrides": {
         "protobufjs": "^7.2.4",
-        "semver": "^7.5.3"
+        "semver": "^7.5.3",
+        "nx": "16.5.5"
```

Comment:
We should not in comments why and when this override is no longer needed if possible

Reply:
any idea of where should I leave those comments? I cannot place them into this json, since is a json

Reply:
What I did before when I needed to leave a comment inside a JSON was to add another field inside the `overrides` object, call it whatever like `note` or `comment` then add the comment right after. like 

```
   "overrides": {
      "protobufjs": "^7.2.4",
        "semver": "^7.5.3",
        "nx": "16.5.5",
        "note": "your_comment",
   }
```

Reply:
Thanks for the idea, and yeah, I thought about it and wanted to put `_comments` for the field name, but since this json is parsed by npm to resolve dependencies, it would think that there is a dep called `_comments` with `my_comment` version, and fail.